shenyu-plugin/shenyu-plugin-api/src/main/java/org/apache/shenyu/plugin/api/utils/WebFluxResultUtils.java

```diff
@@ -72,21 +70,7 @@ public static Mono<Void> result(final ServerWebExchange exchange, final Object r
             .bufferFactory().wrap(bytes))
             .doOnNext(data -> exchange.getResponse().getHeaders().setContentLength(data.readableByteCount())));
     }
-
-    /**
-     * Response result and support specify the http status code .
-     *
-     * @param exchange   the exchange
```

Comment:
why remove this method

Reply:
I encapsulated resultCustomStatusCode by calling the WebFluxResultUtils#result method before. However, the result method will set the MIME type. Although the data of the Mock is a string, it should be set to application/json, so I moved this part to the doExecute of the MockPlugin. The operation of response will be more flexible